pydocumentdb/endpoint_discovery_retry_policy.py

```diff
@@ -40,7 +43,7 @@ def __init__(self, global_endpoint_manager):
         self._max_retry_attempt_count = _EndpointDiscoveryRetryPolicy.Max_retry_attempt_count
         self.current_retry_attempt_count = 0
         self.retry_after_in_milliseconds = _EndpointDiscoveryRetryPolicy.Retry_after_in_milliseconds
-        logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
```

Comment:
Can you also please retain the format of the logger.

Reply:
I didn't do that originally because it adds a little bit of boilerplate, but sure thing. Emulated the full basicConfig code path in bb74225, modeled on the Python 3.6.4 implementation.

Reply:
Ping @srinathnarayanan -- could you please take another look at the pull request?